galileo/src/render/render_bundle/world_set.rs

```diff
@@ -197,7 +197,9 @@ impl WorldRenderSet {
             } => {
                 self.add_shape(point, *fill, *scale, *outline, shape, paint.offset);
             }
-            PointShape::Label { text, style } => self.add_label(point, text, style, paint.offset),
+            PointShape::Label { text, style } => {
+                self.add_label(point, text, style, paint.offset, 1.0)
```

Comment:
If you don't pass dpi parameter here and just use `1.0`, text outline will look too thin on HiDPI monitors.